### PR TITLE
Updated change-log for v1.6.0next

### PR DESCRIPTION
**Pull request by mshariq-nerd**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+* Display the plugin name and version on South and North page
+* Support for password configuration items
+* JSON editor for json data
+* Supprot to update python script 
+* New log tabs for Notification and Packages under logs section
+* Link added in alert to show failure logs 
+
+#### Changed 
+
+* Update asset reading page to show textual data
```
Comment (praveen-garg) [inline docs/changelog.md]: Updated

File: docs/changelog.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+
+#### Fixed
+
+* Fix upload script issue on notification
```
Comment (praveen-garg) [inline docs/changelog.md]: Fixed upload script issue for notification instances

File: docs/changelog.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+* Fixed wrong time on graph data for the past reading
+* Fixed save multiple filter configuration at once
+* Fixed csv formate to download for asset readings
```
Comment (praveen-garg) [inline docs/changelog.md]: Fixed csv formate for asset readings download

File: docs/changelog.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+* Misc performance improvements and visual changes
```
Comment (praveen-garg) [inline docs/changelog.md]: let's remove this.

Comment (praveen-garg) [reply docs/changelog.md]: And add ` Fixed RPM package installation conflict issue`.  I hope we have updated readme i.e. not to use `force` or `replace-file` flag

File: docs/changelog.md (hunk 3 above)
Comment (monika-Dianomic) [reply docs/changelog.md]: csv format

**Pull request merged**